src/webview_provider/types.ts

```diff
@@ -0,0 +1,5 @@
+export type UpdateConfigProps = {
+  key: string;
+  value: string | boolean | number;
+  isPreviewFeature?: boolean;
+};
```

Comment:
If only used in alitmateWebViewProvider. keep it only there and don't export.

Reply:
This is used in datapilot panel as well and will be used in other panels in future if needed

Reply:
but if it is part of the panel it is inherited and will be present on the object that is using it. So I don't immediately see the need to export it and create another file.